webdriver/tests/element_click/link.py

```diff
@@ -0,0 +1,85 @@
+import pytest
+
+from tests.support.asserts import assert_error, assert_success
+from tests.support.inline import inline
+
+def click(session, element):
+    return session.transport.send("POST", "session/{session_id}/element/{element_id}/click"
+                                  .format(session_id=session.session_id,
+                                          element_id=element.id))
+
+
+# 14.1 Element Click Link Element Tests Step 7
+
+def test_click_numbers_link(session):
+    # 14.1 Step 7
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("<a href=%s>123456</a>" % url)
+    element = session.find.css("a", all=False)
+    response = click(session, element)
+    assert_success(response)
+
+    assert session.url == "http://web-platform.test:8000%s" % url
+
+
+def test_click_multi_line_link(session):
+    # 14.1 Step 7
+    url = "/webdriver/tests/element_click/support/input.html"
+    session.url = inline("""
+        <p style="background-color: rgb(255, 255, 0); width: 5em;">
```

Comment:
Avoid `em` since it is in relation to the font the browser is set to use.

Reply:
fixed